src/components/artifacts/artifact-node-sheet/DetailCards.tsx

```diff
@@ -208,14 +209,16 @@ export function DataCard({ artifactVersionId }: Props) {
 						value={
 							<>
 								{isStoreSuccess ? (
-									<Tag
-										emphasis="subtle"
-										rounded={false}
-										color="grey"
-										className="text-theme-text-primary"
-									>
-										{storeData?.name}
-									</Tag>
+									<Link target="_blank" to={artifactStoreHref}>
```

Comment:
Does this intentionally open in a new tab?

I do not have a strong opinion here, but I would have expected navigation inside the same tab

Reply:
this is the same behavior as in pro. you can refer to the PR, where i brought this up for discussion, maybe you missed it?